pkg/tanka/parse.go

```diff
@@ -148,7 +149,13 @@ func ParseEnv(path string, opts ParseOpts) (interface{}, *v1alpha1.Environment,
 		if err != nil {
 			return nil, nil, errors.Wrap(err, "marshalling environment config")
 		}
-		opts.JsonnetOpts.ExtCode.Set(spec.APIGroup+"/environment", string(jsonEnv))
+		// make a copy to prevent race condition in parallel execution
+		injectCode := jsonnet.InjectedCode{}
+		for k, v := range opts.JsonnetOpts.ExtCode {
+			injectCode.Set(k, v)
+		}
+		injectCode.Set(spec.APIGroup+"/environment", string(jsonEnv))
+		opts.JsonnetOpts.ExtCode = injectCode
```

Comment:
InjectedCode.Set caused a race condition in parallel execution, this should prevent that.

```
==================
WARNING: DATA RACE
Write at 0x00c0000dcbc8 by goroutine 40:
  github.com/grafana/tanka/pkg/jsonnet.(*InjectedCode).Set()
      /home/duologic/git/grafana/tanka/pkg/jsonnet/eval.go:24 +0x10d0
  github.com/grafana/tanka/pkg/tanka.ParseEnv()
      /home/duologic/git/grafana/tanka/pkg/tanka/parse.go:153 +0x1039
  github.com/grafana/tanka/pkg/tanka.parseWorker()
      /home/duologic/git/grafana/tanka/pkg/tanka/environments.go:139 +0x23e
  github.com/grafana/tanka/pkg/tanka.ParseParallel.func1()
      /home/duologic/git/grafana/tanka/pkg/tanka/environments.go:93 +0x8f

Previous write at 0x00c0000dcbc8 by goroutine 41:
  github.com/grafana/tanka/pkg/jsonnet.(*InjectedCode).Set()
      /home/duologic/git/grafana/tanka/pkg/jsonnet/eval.go:24 +0x10d0
  github.com/grafana/tanka/pkg/tanka.ParseEnv()
      /home/duologic/git/grafana/tanka/pkg/tanka/parse.go:153 +0x1039
  github.com/grafana/tanka/pkg/tanka.parseWorker()
      /home/duologic/git/grafana/tanka/pkg/tanka/environments.go:139 +0x23e
  github.com/grafana/tanka/pkg/tanka.ParseParallel.func1()
      /home/duologic/git/grafana/tanka/pkg/tanka/environments.go:93 +0x8f

Goroutine 40 (running) created at:
  github.com/grafana/tanka/pkg/tanka.ParseParallel()
      /home/duologic/git/grafana/tanka/pkg/tanka/environments.go:91 +0x178
  github.com/grafana/tanka/pkg/tanka.ExportEnvironments()
      /home/duologic/git/grafana/tanka/pkg/tanka/export.go:58 +0x257
  main.exportCmd.func2()
      /home/duologic/git/grafana/tanka/cmd/tk/export.go:87 +0x6be
  github.com/go-clix/cli.(*Command).execute()
      /home/duologic/gocode/pkg/mod/github.com/go-clix/cli@v0.1.1/command.go:118 +0x45d
  github.com/go-clix/cli.(*Command).Execute()
      /home/duologic/gocode/pkg/mod/github.com/go-clix/cli@v0.1.1/command.go:74 +0x1e4
  main.main()
      /home/duologic/git/grafana/tanka/cmd/tk/main.go:52 +0x3b3

Goroutine 41 (running) created at:
  github.com/grafana/tanka/pkg/tanka.ParseParallel()
      /home/duologic/git/grafana/tanka/pkg/tanka/environments.go:91 +0x178
  github.com/grafana/tanka/pkg/tanka.ExportEnvironments()
      /home/duologic/git/grafana/tanka/pkg/tanka/export.go:58 +0x257
  main.exportCmd.func2()
      /home/duologic/git/grafana/tanka/cmd/tk/export.go:87 +0x6be
  github.com/go-clix/cli.(*Command).execute()
      /home/duologic/gocode/pkg/mod/github.com/go-clix/cli@v0.1.1/command.go:118 +0x45d
  github.com/go-clix/cli.(*Command).Execute()
      /home/duologic/gocode/pkg/mod/github.com/go-clix/cli@v0.1.1/command.go:74 +0x1e4
  main.main()
      /home/duologic/git/grafana/tanka/cmd/tk/main.go:52 +0x3b3
==================
```